src/Main.hs

```diff
@@ -141,6 +131,9 @@ intersects (Ray origin direction) object@(Object (Sphere center radius) material
             [] -> Nothing
             otherwise -> Just (minimum listOfRoots, object)
 
+deg2Rad :: Double -> Double
```

Comment:
possibly better to move this into a Util module. could also just pull in a lib, or just have some way of forcing use of radians by default in the camera